src/CalcManager/CEngine/scicomm.cpp

```diff
@@ -103,6 +103,14 @@ void CCalcEngine::ProcessCommandWorker(OpCode wParam)
         m_nTempCom = (int)wParam;
     }
 
+    // Clear expression shown after = sign, when user do any action.
+    static bool afterEQUCommand = false;
+    if (afterEQUCommand)
+    {
+        afterEQUCommand = false;
+        m_HistoryCollector.ClearHistoryLine(wstring());
```

Comment:
NIT: the history has already been CompleteHistoryLine`, we only need to clear ExpressionDisplay (what <kbd>=</kbd> currently does)

```cpp
 if (nullptr != m_pCalcDisplay)	
{	
   m_pCalcDisplay->SetExpressionDisplay(	
       make_shared<CalculatorVector<pair<wstring, int>>>(), 
       make_shared<CalculatorVector<shared_ptr<IExpressionCommand>>>());	
}
```